lib/single-new-eks-cost-monitoring-pattern/kubecostserviceaccountsaddon.ts

```diff
@@ -0,0 +1,72 @@
+import 'source-map-support/register';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import * as eks from "aws-cdk-lib/aws-eks";
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import { Construct } from 'constructs';
+
+
+export class KubecostServiceAccountsAddon implements blueprints.ClusterAddOn {
```

Comment:
Why is this required? Please remove. It should work with Kubecost free version same as blog.

Reply:
Yeah this is just creating service accounts required with the right IAM roles to access Prometheus